songs_app_backend/songs/models.py

```diff
@@ -0,0 +1,72 @@
+from django.db import models
+from taggit.managers import TaggableManager
+from users.models import User
+from datetime import datetime
+# Create your models here.
+
+
+class Song(models.Model):
+    title = models.CharField(max_length=100, unique=True)
+    singer = models.CharField(max_length=100)
+    tags = TaggableManager()
+    created_time = models.DateTimeField()
```

Comment:
for created_at there's another attribute. `created_at = models.DateTimeField(auto_now_add=True)`
This will automatically save the time and date when the song is created and you don't have to do anything.